rust/main/hyperlane-base/src/settings/parser/mod.rs

```diff
@@ -332,9 +332,16 @@ fn parse_signer(signer: ValueParser) -> ConfigResult<SignerConf> {
                 .get_key("prefix")
                 .parse_string()
                 .unwrap_or_default();
+            let account_id_type = signer
+                .chain(&mut err)
+                .get_opt_key("account_id_type")
+                .parse_string()
+                .end()
+                .unwrap_or("BITCOIN");
```

Comment:
nit: to get feedback on the parsing here instead of later on, I think we could have the `account_id_type` on `SignerConf::CosmosKey` be a `AccountIdType` directly instead of a `String`, and then we can do
```
let account_id_type = signer
                .chain(&mut err)
                .get_opt_key("account_id_type")
                .parse_from_string()
                .unwrap_or(AccountIdType::Bitcoin);
```
to parse into the enum here.

Similar to how it works up above for the AWS region. I think it may require `FromStr`/`str::parse` on `AccountIdType`

Reply:
Fixed